src/screens/app/Components/UserCaption/UserCaption.js

```diff
@@ -1,5 +1,5 @@
+/* eslint-disable react-native/no-inline-styles */
```

Comment:
Ensure this file doesn't conflict with what's on master. Otherwise remove commit.